packages/mobile/src/components/comments/CommentBlock.tsx

```diff
@@ -55,7 +57,11 @@ export const CommentBlockInternal = (
   } = comment
   const isTombstone = 'isTombstone' in comment ? !!comment.isTombstone : false
   const isPinned = track.pinned_comment_id === commentId
+  const isHighlighted = highlightCommentId === commentId
 
+  const { focus } = useThemeColors()
```

Comment:
lets use `useTheme` for both colors and theme variant, i believe "theme.type" is the variant, and likely there is the right color in "theme.color...."